core/src/main/java/software/amazon/awssdk/util/DateUtils.java

```diff
@@ -34,29 +40,51 @@
  */
 @ThreadSafe
 public class DateUtils {
-    private static final DateTimeZone GMT = new FixedDateTimeZone("GMT", "GMT", 0, 0);
-    /** ISO 8601 format. */
-    protected static final DateTimeFormatter ISO_8601_DATE_FORMAT =
-            ISODateTimeFormat.dateTime().withZone(GMT);
     /** Alternate ISO 8601 format without fractional seconds. */
-    protected static final DateTimeFormatter ALTERNATE_ISO_8601_DATE_FORMAT =
-            DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss'Z'").withZone(GMT);
+    static final DateTimeFormatter ALTERNATE_ISO_8601_DATE_FORMAT =
+        new DateTimeFormatterBuilder()
+            .appendPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
+            .toFormatter()
+            .withZone(UTC);
+
+    private static final int AWS_DATE_MILLI_SECOND_PRECISION = 3;
+
+    public static final DateTimeFormatter ISO_8601_DATETIME_FORMAT =
```

Comment:
I wanted to preserve the format as much as possible. I will switch to ISO_INSTANT and add another test with fractional seconds to ensure backwards compatibility.